SoftwareGuide/Latex/Appendices/CodingStyleGuide.tex

```diff
@@ -954,8 +954,7 @@ \subsubsection{Variable Initialization}
 \begin{minted}[baselinestretch=1,fontsize=\footnotesize,linenos=false,bgcolor=ltgray]{cpp}
 template <typename TInputImage>
 double
-WarpHarmonicEnergyCalculator<TInputImage>
-::EvaluateAtNeighborhood(ConstNeighborhoodIteratorType & it) const
+WarpHarmonicEnergyCalculator<TInputImage>::EvaluateAtNeighborhood(ConstNeighborhoodIteratorType & it) const
```

Comment:
I am almost sure this will go over the side of the page.

Reply:
Agreed.  I've run it all again using the `.clang_format` file from `ITK/Examples`.  This is still a DRAFT.